e2e-tests/pipeline_validation.sh

```diff
@@ -40,13 +40,16 @@ function usage() {
   echo "                              or host network URLs"
   echo "     --streaming              Flag to specify whether we are testing a"
   echo "                              streaming pipeline"
+  echo "     --jdbc_mode_hapi         Flag to specify whether we are testing a"
```

Comment:
Should this be something more generic like `--hapi` or is there anything specific about the JDBC mode?